test/lens_test.exs

```diff
@@ -37,6 +37,27 @@ defmodule LensTest do
     end
   end
 
+  describe "otherwise" do
+    test "to_list" do
+      lens = Lens.empty() |> Lens.otherwise(Lens.const(:none))
+
+      assert Lens.to_list(lens, [:a, :b]) == [:none]
+
+      lens = Lens.otherwise(Lens.indices([0, 2]), Lens.back())
+
+      assert Lens.to_list(lens, [2, 3, 4]) == [2, 4]
+    end
+
+    test "map" do
```

Comment:
This case is already included in the docs, should run as part of doctests

Reply:
I mean that a very similar test case to this `test "map"` is already included in the doctests, so it should not be replicated here